redis-hbo-provider/src/main/java/com/facebook/presto/statistic/RedisProviderPlugin.java

```diff
@@ -34,7 +34,6 @@ public class RedisProviderPlugin
 
     private Injector injector;
 
-    @VisibleForTesting
     public RedisProviderPlugin(String configPath)
```

Comment:
also, can't comment on the line and it's not related to this fix, but why do we set the property map to null if we can't load the properties?  I'd expect that if someone set the properties, they would be surprised to discover they don't really have an hbo provider set up, so I think it would be better to throw an exception.

Reply:
Agreed